### ISSUE TITLE
The version of Microsoft.NET.Sdk used by this project is insufficient to support references to libraries targeting .NET Standard 1.5 or higher

### ISSUE THREAD

**Issue by ccalvarez** [aspnet/Caching#355]
Hi.

I have cloned this repo, ran the `build.cmd /t:Restore` command, and opened solution with Visual Studio 2017.  When I build the solution, or any project separately, I get this errors:

1. **The version of Microsoft.NET.Sdk used by this project is insufficient to support references to libraries targeting .NET Standard 1.5 or higher.  Please install version 2.0 or higher of the .NET Core SDK.**	SqlServerCacheSample	C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\Microsoft\Microsoft.NET.Build.Extensions\Microsoft.NET.Build.Extensions.NETFramework.targets	

2. **The current .NET SDK does not support targeting .NET Core 2.0.  Either target .NET Core 1.1 or lower, or use a version of the .NET SDK that supports .NET Core 2.0.**	Microsoft.Extensions.Caching.SqlServer.Tests	C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.TargetFrameworkInference.targets	

**My environment is:**

- OS:  Windows 10 Pro
- OS Version:  1703
- OS Build:  15063.632
- Visual Studio Enterprise 2017 Version 15.3.5

**- dotnet --info ran on c:\:**
```
C:\
λ dotnet --info
.NET Command Line Tools (2.0.0)

Product Information:
 Version:            2.0.0
 Commit SHA-1 hash:  cdcd1928c9

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.15063
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.0.0\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0
  Build    : e8b8861ac7faf042c87a5c2f9f2d04c98b69f28d
```

**- dotnet --info ran on repository folder:**
```
λ dotnet --info
The specified SDK version [2.1.0-preview1-007042] from global.json [C:\folder\subfolder\Caching\global.json] not found; install specified SDK version

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0
  Build    : e8b8861ac7faf042c87a5c2f9f2d04c98b69f28d
```

I have also added C:\Users\myusername\\.dotnet folder to PATH system environment variable.

I'd appreciate your help, I need to be able to rebuild this libraries.  Thanks.

**Comment by Tratcher** [comment]
@jkotalik @natemcmaster

**Comment by ccalvarez** [comment]
@jkotalik @natemcmaster @Tratcher:

Thanks! 

I've just found the solution:

What I understand is that I had installed .NET Core 2.0 SDK, and since I had the folder `c:\Program Files\dotnet` in the PATH system environment variable, my Visual Studio always detected that SDK instead of SDK Version 2.1.0-preview1-007042 stated in global.json file and installed with the `build.cmd` command.

What I have done was to remove `c:\Program Files\dotnet` and  `C:\Users\myusername\.dotnet` from PATH, and add `C:\Users\myusername\.dotnet\x64` to the PATH.

So now when I run `dotnet --info` withing the repository folder I get:

```
λ dotnet --info
.NET Command Line Tools (2.1.0-preview1-007042)

**Product Information:
 Version:            2.1.0-preview1-007042**
 Commit SHA-1 hash:  e94a82c9a3

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.15063
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Users\ccord\.dotnet\x64\sdk\2.1.0-preview1-007042\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0
  Build    : e8b8861ac7faf042c87a5c2f9f2d04c98b69f28d
```

And now I am able to build the projects with Visual Studio 2017.

Thank you very much!
